.eslintrc

```diff
@@ -13,7 +13,29 @@
     "react"
   ],
   "ecmaFeatures": {
-    "jsx": true
+    "jsx": true,
+    "arrowFunctions": true,
+    "binaryLiterals": true,
+    "blockBindings": true,
+    "classes": true,
+    "defaultParams": true,
+    "destructuring": true,
+    "forOf": true,
+    "generators": true,
```

Comment:
I'm interested in the rationale for disallowing any ES2015 features. In my experience generators are a very useful abstraction.